### PR TITLE
Emerald Pouch Redirects

### PR DESCRIPTION
**Pull request by ryanzhoudev**

### CODE REVIEW

**Comment by Incompleteusern** [comment]
GameUpdateOverlay#queueEditableMessage seems to be the same as GameUpdateOverlay#queueMessage but it returns the mssage. Since the editing functionality exists for all MessageContainers, you can merge the queueEditableMessage into queueMessage and just return it, as it is redundant currently. 

I like the implementation as it is really clean though.

**Comment by ryanzhoudev** [comment]
> GameUpdateOverlay#queueEditableMessage seems to be the same as GameUpdateOverlay#queueMessage but it returns the mssage. Since the editing functionality exists for all MessageContainers, you can merge the queueEditableMessage into queueMessage and just return it, as it is redundant currently. 
> 
> I like the implementation as it is really clean though.

I think I tried merging both functions together, and it raised '88 related problems' in IDEA. I don't remember what they were though, I'll take a look in a bit

Edit - Done!

File: src/main/java/com/wynntils/modules/utilities/configs/OverlayConfig.java [hunk 1]
```diff
@@ -355,8 +355,11 @@
             @Setting(displayName = "Redirect AFK Messages", description = "Should messages about AFK Protection be redirected to the game update ticker?")
             public boolean redirectAfk = true;
 
-            @Setting(displayName = "Redirect pouch Messages", description = "Should messages about ingredients being added to your pouch be redirected to the game update ticker?")
-            public boolean redirectPouch = true;
+            @Setting(displayName = "Redirect Ingredient Pouch Messages", description = "Should messages about ingredients being added to your pouch be redirected to the game update ticker?")
+            public boolean redirectIngredientPouch = true;
+
+            @Setting(displayName = "Redirect Emerald Pouch Messages", description = "Should messages about ingredients being added to your pouch be redirected to the game update ticker?")
```
Comment (magicus) [inline src/main/java/com/wynntils/modules/utilities/configs/OverlayConfig.java]:
```suggestion
            @Setting(displayName = "Redirect Emerald Pouch Messages", description = "Should messages about emeralds being added to your pouch be redirected to the game update ticker?")
```

**Review by magicus — CHANGES_REQUESTED** [review]

**Review by magicus — APPROVED** [review]

**Comment by ryanzhoudev** [comment]
If these changes are okay and EmeraldPouchManager.java gets to stay, the usage arc #352 can be updated with those methods, will do that in a PR after all these are merged

**Review by magicus — APPROVED** [review]
I still think this looks good. I'd like to see some other developer chime in (or commit) though, since I'm not so active in the code base right now.

**Pull request merged**